.pre-commit-config.yaml

```
@@ -155,7 +155,7 @@ repos:
     additional_dependencies:
     - types-docutils
     - lxml  # dep of `--txt-report`, `--cobertura-xml-report` & `--html-report`
-    - pytest
+    - pytest<8
```

match the surrounding style
```suggestion
    - pytest < 8
```